### DISCUSSION TITLE
yalmip can save the “gap” and “time” informations for every loop？

### DISCUSSION THREAD

**Discussion by lzmssyld** [yalmip/YALMIP#1438] [General discussion] [answered]
Dear professor.
Actually, I have two questions:
1.I want to change the parameters of the model and solve them in a loop. What command do I need  to record the time and gap information for each solution?Yalmip will output them directly in the command line window，like this:

    Nodes    |    Current Node    |     Objective Bounds      |     Work
 Expl Unexpl |  Obj  Depth IntInf | Incumbent    BestBd   Gap | It/Node Time

     0     0 942482.000    0   76 6269648.00 942482.000  85.0%     -    3s
     0     0 942482.000    0   75 6269648.00 942482.000  85.0%     -    3s
H    0     0                    6209882.0000 942482.000  84.8%     -    6s
     0     0 942482.000    0 2822 6209882.00 942482.000  84.8%     -   36s
     0     0 942482.000    0 2338 6209882.00 942482.000  84.8%     -   36s
H    0     0                    6196007.0000 942482.000  84.8%     -   64s
     0     0 942482.000    0 1544 6196007.00 942482.000  84.8%     -   67s

2.Can yalmip achieve  the function in the following website? Use 'callback' to set multiple termination criteria.
https://support.gurobi.com/hc/en-us/articles/360013419411-How-do-I-set-multiple-termination-criteria-for-a-model

**Comment by johanlofberg** [comment, marked as answer]
1 https://yalmip.github.io/Extracting
2 Doubt they support it in matlab

**Reply by lzmssyld** [reply]
For question 2，you think gurobi doesn,t support that function  in matlab ?

**Reply by johanlofberg** [reply]
correct

**Reply by lzmssyld** [reply]
You are right.Thanks for your paitence.
https://support.gurobi.com/hc/en-us/community/posts/360073853312-How-to-add-Lazy-Constraints-and-Callbacks-in-MATLAB
